api/app/signals/apps/email_integrations/settings.py

```diff
@@ -0,0 +1,53 @@
+from django.conf import settings
+from django.test.signals import setting_changed
+
+DEFAULTS = {
```

Comment:
This cannot be a `set` literal (see line 29).

Reply:
Changed